contrib/bdii/htcondor-ce-provider

```diff
@@ -274,13 +274,19 @@ def get_cert_issuer(path):
 
 
 def get_ce_jobs(ce_schedd_ad, time_out):
-    ce_schedd = htcondor.Schedd(ce_schedd_ad)
+    try:
+        ce_schedd = htcondor.Schedd(ce_schedd_ad)
+    except RuntimeError as exc:
+        sys.stderr.write("%s\n" % exc)
```

Comment:
```suggestion
        sys.stderr.write("%s: %s\n" % (exc, ce_fqdn))
```

Not sure if the exception contains the CE FQDN so we may as well include it like below

Reply:
This isn't going to work as suggested because `ce_fqdn` isn't initialized yet. We can easily bump it up to the top line. This also raises the question: should the `htcondor.Schedd()` and `ce_schedd.xquery()` calls go in the same try/catch since they both catch exceptions exactly the same way?

Reply:
Good catch. That all sounds good to me!

Reply:
Great! It's all done, going to merge now.